packages/orchestrator-ui-components/src/utils/date.ts

```diff
@@ -1,3 +1,5 @@
+import { Locale } from '../types/types';
```

Comment:
A better naming of the types folder or the files inside could be proposed, so they don't clash.

Reply:
I have no better suggestions :(. Are you sure you can't use the Locale from the package types?

Reply:
Yep, it fails the unit-tests if use '@/types'